src/judging_round_selector.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Form, Select } from 'semantic-ui-react'
+
+
+class JudgingRoundSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            user: []
+        };
+    }
+
+    componentDidMount() {
+        fetch("http://localhost:8000/api/v1/judging_round/?round_type=Online",
```

Comment:
Same issue as above, literal and hardcoded reference to localhost

Reply:
Same comment.